lms/envs/common.py

```diff
@@ -1145,6 +1145,10 @@ def _make_mako_template_dirs(settings):
 SESSION_COOKIE_DOMAIN = ""
 SESSION_COOKIE_NAME = 'sessionid'
 
+# django-session-cookie middleware
```

Comment:
Should we add options to override these from `lms.env.json` in case people want to use "Strict" or "Lax" depending on their use cases?

Reply:
@kaizoku  @pomegranited It does look like the `django_cookies_samesite` middleware looks for the `SESSION_COOKIE_NAME` setting here.
https://github.com/jotes/django-cookies-samesite/blob/v0.5.1/django_cookies_samesite/middleware.py#L54

It throws and error if that setting is not found here.
https://github.com/jotes/django-cookies-samesite/blob/v0.5.1/django_cookies_samesite/middleware.py#L77-L79

I've place this in my `edx-platform/lms/envs/private.py` file currently but I could see adding this also to `lms.env.json` by adding the `SESSION_COOKIE_NAME` setting `edx-platform/lms/envs/production.py` here.
https://github.com/open-craft/edx-platform/blob/jill/samesite-cookies/lms/envs/production.py#L209-L213

Reply:
@kaizoku -- good point.  I'll make this configurable for the production and devstack envs.

Reply:
@ztraboo -- [`SESSION_COOKIE_NAME` was added to lms.envs.common](https://github.com/edx/edx-platform/commit/4a93420946aa7c928a625148c1e9d831afb35947#diff-a8eedcb5656641698b92541a82609962R784), so we don't need to set this specially anymore.